util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+
+# Output list of possible mount points
+_fscrypt_mountpoints()
+{
+    fscrypt status 2>/dev/null | \
+        command sed \
+            -e '/^\//!d' \
+            -e '/Yes$/!d' \
+            -e 's/ .*//'
+}
+
+# Output list of possible policy or protector IDs
+# $1: the mount point on which policies are looked for.
+# $2: the section (policy or protector) to retrieve
+_fscrypt_status_section()
+{
+    fscrypt status "$1" 2>/dev/null | \
+        command sed \
+            -e "/^${2^^} /",'/^$/!d' \
+            -e "s/${2^^}//" \
+            -e "s/\s.*//"
+}
+
+_fscrypt()
+{
+    local cur prev words cword split
+    local GLOBAL_OPTS="--verbose --quiet --help"
+    _init_completion -s -n : || return
+
+    case $prev in
+        --drop-caches)
```

Comment:
`--drop-caches` is a no-op except for v1 encryption policies, which are deprecated.  So `--drop-caches` is effectively deprecated too.  I recommend omitting it from the bash completion support.

Reply:
Yes, we shouldn't propose use of deprecate things.  I didn't know that `--drop-caches` was.

Dropped.

Reply:
Looks like `--drop-caches` didn't get fully removed.

Reply:
Oops, I missed some of them…